doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -95,5 +95,14 @@ While this Reference Architecture is being developed, Gaps will be identified th
 <a name="6.3"></a>
 ## 6.3 Proposals & Resolution
 
+### 6.3.2 Multi-tenancy and workload isolation with Kubernetes
+
+Kubernetes is not a single cluster solution. This has been demonstrated across the industry from case studies at prominent companies like [Twitter](https://www.alibabacloud.com/blog/what-can-we-learn-from-twitters-move-to-kubernetes_595156), [USA Today](https://medium.com/usa-today-network/there-and-back-again-scaling-multi-tenant-kubernetes-cluster-s-67afb437716c), [Zalando](https://www.youtube.com/watch?v=LpFApeaGv7A), and [Alibaba](https://www.cncf.io/blog/2019/12/12/demystifying-kubernetes-as-a-service-how-does-alibaba-cloud-manage-10000s-of-kubernetes-clusters/) to the bi-annual CNCF survey that finds that the number of clusters being deployed within an organization is growing. While there are many reasons behind the multi cluster paradigm, examining the gap above we find that a multi cluster solution can address many of these problems like security and software life cycle management. 
+
+Without hard multi tenancy within a clusters, separate clusters must be used to provide adequate separation for different types of workloads like production vs. non production or per categorization. Having multiple clusters to deploy CNFs into allows operators to chose similar CNFs together while segregating those with different lifecycles from each other. CNFs deployed into the same cluster can be upgraded together to reduce the operational load while CNFs that require different versions, configurations, and dependencies can run in separate clusters and be upgraded independently.
```

Comment:
I think in our industry the main question is the execution of CNF-s from different vendors in the same Kubernetes cluster.

Reply:
is there different wording you would like or something you would like to add?

Reply:
Maybe extend this part "separation for different types of workloads like production vs. non production or per categorization." to something like "separation for different types of workloads based on their vendors, type like production vs. non production or per categorization.".

Reply:
Thanks just updated it